packages/office-ui-fabric-react/src/components/Announced/examples/Announced.SearchResults.Example.tsx

```diff
@@ -31,13 +34,14 @@ export class AnnouncedSearchResultsExample extends React.Component<
   IAnnouncedSearchResultsExampleProps,
   IAnnouncedSearchResultsExampleState
 > {
-  private timer: number;
+  private timer: NodeJS.Timer;
```

Comment:
Hrmm `NodeJS.Timer` is the incorrect type. `tsc` may be complaining this is the wrong type because it does not understand it is in the browser context. The type you had previously `number`, is correct.

https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/setInterval#Return_value

We should fix these `NodeJS.Timer` types prior to merge.